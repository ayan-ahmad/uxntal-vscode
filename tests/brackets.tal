[ #00 ]
[[ #00 ]]
[x #00 ]x
x[x #00 x]x
