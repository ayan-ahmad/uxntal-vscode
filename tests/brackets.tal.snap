>[ #00 ]
#^ source.tal punctuation.brackets.uxntal
# ^ source.tal
#  ^ source.tal punctuation.rune.uxntal
#   ^^ source.tal constant.numeric.uxntal
#     ^ source.tal
#      ^ source.tal punctuation.brackets.uxntal
>[[ #00 ]]
#^^ source.tal variable.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.rune.uxntal
#    ^^ source.tal constant.numeric.uxntal
#      ^ source.tal
#       ^^ source.tal variable.uxntal
>[x #00 ]x
#^^ source.tal variable.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.rune.uxntal
#    ^^ source.tal constant.numeric.uxntal
#      ^ source.tal
#       ^^ source.tal variable.uxntal
>x[x #00 x]x
#^^^ source.tal variable.uxntal
#   ^ source.tal
#    ^ source.tal punctuation.rune.uxntal
#     ^^ source.tal constant.numeric.uxntal
#       ^ source.tal
#        ^^^ source.tal variable.uxntal
>